tools/gen-intrinsics-specs.py

```diff
@@ -740,9 +742,14 @@ def process_db(db, classification_db):
     assert(table_header is not None)
     body = ""
     for k, v in filtered.items():
-        body += "\n" + \
-            recurse_print_to_rst((k, v), rst_header_levels,
-                                 headers=table_header, tablefmt="grid")
+        if file_format=="rst":
+            body += "\n" + \
+                recurse_print_to_rst((k, v), rst_header_levels,
+                                     headers=table_header, tablefmt="grid")
+        elif file_format=="md":
+            body += "\n" + \
+                recurse_print_to_rst((k, v), rst_header_levels,
+                                     headers=table_header, tablefmt="github")
```

Comment:
Please void having 2 invocation of the same method. Just lift the values `github` and `grid` in the shell script (replacing `md` and `rst` respectively) and pass them all the way down here, so you can just have one invocation of `recurse_print_to_rst`